### PR TITLE
Redis Lock Facility

### PR DESCRIPTION
**Pull request by rbndg**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  _stop (cb) {
+    async.series([
+      next => { super._stop(next) },
+      next => {
+        this.redisLock.quit(next)
+      },
+      next => {
+        delete this.redisLock
+        delete this.redisCluster
```
Comment (robertkowalski) [inline index.js]: this can get removed i think

**Comment by robertkowalski** [comment]
can you add a few docs how to setup teh fac togetehr with the redis client?

File: index.js (hunk 1 above)
Comment (rbndg) [reply index.js]: yes, good catch

**Comment by rbndg** [comment]
Added some documentation

**Comment by robertkowalski** [comment]
<3 

thanks, great stuff!

**Comment by rbndg** [comment]
@prdn changed redisLock to lock

**Pull request merged**